package.json

```diff
@@ -14,48 +14,51 @@
   },
   "dependencies": {
     "@expo/vector-icons": "^13.0.0",
+    "react-native-markdown-display": "ProgPrak-Native-App/react-native-markdown-display",
     "@js-joda/core": "^5.2.0",
     "@react-native-async-storage/async-storage": "~1.17.3",
     "@react-native-community/datetimepicker": "6.1.2",
     "@react-native-community/slider": "4.2.1",
     "@react-navigation/bottom-tabs": "^6.3.1",
     "@react-navigation/native": "^6.0.10",
     "@react-navigation/native-stack": "^6.6.2",
+    "@types/react-native-snap-carousel": "^3.8.5",
     "expo": "^45.0.0",
     "expo-av": "~11.2.3",
     "expo-checkbox": "~2.1.0",
-    "expo-clipboard": "~3.0.1",
-    "expo-haptics": "~11.2.0",
     "expo-media-library": "~14.1.0",
     "expo-notifications": "~0.15.4",
     "expo-permissions": "~13.2.0",
-    "expo-secure-store": "~11.2.0",
     "expo-splash-screen": "^0.15.1",
     "expo-status-bar": "^1.3.0",
-    "expo-updates": "~0.13.3",
+    "expo-updates": "^0.13.2",
+    "webpack-dev-server": "~3.11.0",
+    "@expo/webpack-config": "~0.16.2",
+    "expo-secure-store": "~11.2.0",
     "nanoid": "^4.0.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-native": "~0.68.2",
-    "react-native-calendars": "^1.1285.0",
+    "node-fetch": "^3.2.6",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "react-native": "^0.68.2",
+    "react-native-calendars": "^1.1275.0",
     "react-native-countdown-circle-timer": "^3.0.9",
     "react-native-countdown-component": "^2.7.1",
     "react-native-get-random-values": "~1.8.0",
-    "react-native-markdown-display": "ProgPrak-Native-App/react-native-markdown-display",
     "react-native-safe-area-context": "4.2.4",
+    "react-native-screens": "~3.11.1",
     "react-native-snap-carousel": "^4.0.0-beta.6",
     "react-native-sortable-list": "^0.0.25",
-    "react-native-web": "~0.17.7"
+    "react-native-web": "0.17.7",
+    "expo-clipboard": "~3.0.1",
+    "expo-haptics": "~11.2.0"
```

Comment:
Die Dependency-Änderungen sind versehentlich, oder?
Sowas wie `@types/*`, `webpack-dev-server` oder `@expo/webpack-config` gehört jedenfalls in die devDependencies